test/cert-ceremonies/root-ceremony-rsa.yaml

```diff
@@ -20,3 +20,14 @@ certificate-profile:
     key-usages:
         - Cert Sign
         - CRL Sign
+skip-lints:
+   - e_ext_authority_key_identifier_missing
+   - e_ext_authority_key_identifier_no_key_identifier
+   - e_sub_ca_aia_does_not_contain_ocsp_url
+   - e_sub_ca_aia_missing
+   - e_sub_ca_certificate_policies_missing
+   - e_sub_ca_crl_distribution_points_missing
+   - n_ca_digital_signature_not_set
```

Comment:
The digitalSignature EKU is only required on root CAs if they will be directly issuing OCSP responses, which our roots don't.